IFC4x3/PropertyEnumerations/b/PEnum_BumperOrientation.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_BumperOrientation" Name="PEnum_BumperOrientation" UniqueId="7cc5c059-92fa-45d6-96c2-dad7c16176ed">
+	<Documentation>Enumeration indicating the orientation of a bumper element</Documentation>
```

Comment:
Probably better to do this in IfcDoc. It should generate an md file.

Reply:
Not sure why it was like this. I just removed its documentation, not necessary.

Reply:
Alright. If this documentation is not needed, I will merge.